docs/setup.md

```diff
@@ -56,7 +56,11 @@ custom_js_with_timestamps:
       <h2><span class="step-no">4</span> Create <code>.babelrc</code> configuration file</h2>
 
       <p>Great! You've configured Babel but you haven't made it actually <em>do</em> anything. Create a <a href="/docs/usage/babelrc">.babelrc</a> config in your project root and enable some <a href="/docs/plugins">plugins</a>.</p>
-      
+
     </div>
+
+    <p>
+      <strong>Note:</strong>: Running a Babel 6.x project using npm 2.x can cause performance problems because of the way npm 2.x install's dependencies. This problem can be eliminated by either switching to npm 3.x or running npm 2.x with the <a href="https://docs.npmjs.com/cli/dedupe">dedupe</a> flag. To check what version of npm you have run `npm --version`
```

Comment:
Typos: there are 2 colons (`:`). I think `install's dependencies` should be `installs dependencies` right?

Since this is html you will have to use `<pre>` instead of backticks for `npm --version`